virt/virt-2-4-release-notes.adoc

```diff
@@ -24,6 +24,17 @@ include::modules/virt-what-you-can-do-with-virt.adoc[leveloffset=+2]
 //CNV-5438 - svirt-support-for-cnv
 * This release features significant security enhancements. {VirtProductName} now supports SELinux with Mandatory Access Control (MAC) for isolating virtual machines (VMs). Previously, all VMs were managed by using privileged xref:../security/container_security/security-platform.adoc#security-deployment-sccs_security-platform[Security Context Constraints (SCC)]. Now, you can use less privileged custom SCCs for VMs and limit the use of privileged SCCs to infrastructure containers in the cluster.
 
+//CNV-4572-Guest-OS
+[id="virt-guest-os-new"]
+=== Supported guest operating systems
+{VirtProductName} guests can use the following operating systems:
+
+** Red Hat Enterprise Linux 6, 7, 8.
+** Microsoft Windows Server 2012 R2, 2016, and 2019.
+** Microsoft Windows 10.
++
```

Comment:
Please remove the `+` so that this statement aligns with the intro sentence (now that the initial bullet point has been removed).

Reply:
@ousleyp Request clarification. I didn't understand this comment. Please show the actual structure of you want it. Thanks.

Reply:
For both of these comments, please see if the asciidoc reference clarifies things: https://asciidoctor.org/docs/asciidoc-syntax-quick-reference/#lists

Regarding the `+`, see the "list continuation" example in the section titled "Complex content in outline lists" and let me know if you still have questions. Thanks!

Reply:
Content was updated.